src/swaps/bitcoinv2.rs

```diff
@@ -437,12 +523,13 @@ impl BtcSwapTxV2 {
         }
     }
 
-    /// Compute the Musig partial signature for Submarine Swap.
-    /// This is used to cooperatively close a submarine swap.
-    pub fn submarine_partial_sig(
+    /// Compute the Musig partial signature.
+    /// This is used to cooperatively close a Submarine or Chain Swap.
+    pub fn partial_sig(
```

Comment:
```suggestion
    pub fn partial_sign(
```
Just `partial_sig` sounds a little ambiguous. Feel free to leave it as is if you think the comment is clear enough